src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
```

Comment:
is this suppose to be a unidirectional or bidirectional mapping?

if bidirectional - then you need to add a reference to actors in `Country` class

if unidirectional - you may skip this comment

Reply:
I think from the point of business logic, countries don't have to know which actors were born in them, so I will leave this moment unchanged.